.babelrc

```diff
@@ -1,3 +1,21 @@
 {
-  "presets": ["es2015", "stage-2", "react"]
+  "presets": ["react"],
+  "env": {
+    "commonjs": {
+      "plugins": [
```

Comment:
Actually after some tests, `.babelrc` is working as expected, but the `main` on package.json is pointing to the non-compiled version. You change to compile from `src` to `lib` should fix without to much change (I believe).